pom.xml

```diff
@@ -13,4 +14,105 @@
 		<module>game-server</module>
 		<module>login-server</module>
 	</modules>
-</project>
\ No newline at end of file
+
+	<properties>
+		<maven.compiler.release>21</maven.compiler.release>
+		<maven.test.skip>true</maven.test.skip>
+
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+	</properties>
+
+	<dependencyManagement>
+		<dependencies>
+			<dependency>
+				<groupId>com.aionemu</groupId>
+				<artifactId>commons</artifactId>
+				<version>${project.version}</version>
+			</dependency>
+		</dependencies>
+	</dependencyManagement>
+
+	<build>
+		<pluginManagement>
+			<plugins>
+				<plugin>
+					<groupId>org.apache.maven.plugins</groupId>
+					<artifactId>maven-compiler-plugin</artifactId>
+					<version>3.13.0</version>
+					<configuration>
+						<compilerArgs>
+							<arg>-Xlint:all,-preview,-this-escape</arg>
+						</compilerArgs>
+						<debug>true</debug>
```

Comment:
removed the `showWarnings` ([defaults to `true`](https://maven.apache.org/plugins/maven-compiler-plugin/compile-mojo.html#showWarnings)), but left the `debug` ([defaults to `false`](https://maven.apache.org/plugins/maven-compiler-plugin/compile-mojo.html#debug)) arguments.

Reply:
My IDE tooltip says `debug` defaults to `true`, as well as the documentation here: https://maven.apache.org/plugins/maven-compiler-plugin/compile-mojo.html#debug 🤔

Reply:
huh, ok. but then again "set to true to debug" xD that makes no sense. anyway. I'll trust the doc and change it.

Reply:
Yeah, the description indicates otherwise. Maybe the default value was false in an earlier version 🤷‍♂️

Reply:
did adjust it.